### ISSUE TITLE
Vacation filter: Button to add recipients (+) does nothing

### ISSUE THREAD

**Issue by cm-schl** [the-djmaze/snappymail#728]
**Describe the bug**
When you try to create a vacation filter in the SIEVE configuration you can see a (+) button beside the input field.
This button seems to do nothing.

**Expected behavior**
The mail addresses of the user should be inserted into the field eventually with possibility to choose which addresses to add.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/63400209/204782848-e80ab662-c670-44d5-879d-280f26b48479.png)

**Additional context**
Looking at the code an https://github.com/the-djmaze/snappymail/commit/d6dc4d291c7c87f618ef408c309beebcd86f795f the line seems to be commented out https://github.com/the-djmaze/snappymail/blob/9df052a435632bf6825046fa5bb8aec965f0b1d6/dev/Sieve/Model/Filter.js#L214

**Comment by cm-schl** [comment]
Thanks for the update - not directly that what I expected, but I'm ok with that 😅.
Just a question if maybe someone would need it: is getting a list of mail addresses of the user not possible and was this therefore disabled?

**Comment by the-djmaze** [comment]
Correct. Sieve filter editor is moved to separate script and therefore has no access to user accounts.
It was a silly option anyway because it only works for own multi-account setups.
